oat-sa/extension-tao-qtiprint
# Fix BSA-203 - Add css styles for choice interaction with horizontal position

Related to: https://oat-sa.atlassian.net/browse/BSA-203

Added css rules for choice interaction with a horizontal position

How to test:
1. install `wkhtmltopdf` https://hub.taotesting.com/techdocs/tao-booklet/readme-booklet
2. set `define('DEBUG_MODE', false);` in `config/generis.conf.php`
3. in `config/taoBooklet/wkhtmltopdf.conf.php` set binary to wkhtmltopdf path for example `'binary' => '/usr/local/bin/wkhtmltopdf',`
3. compile bundles for taoQtiPrint and taoBooklet
`cd tao/views/build`
`npx grunt taoqtiprintbundle` 
`npx grunt taobookletbundle` 
4. create items **with horizontal choice interaction**
5. create test with these items
6. generate booklet from test
7. open booklet in preview and download pdf
8. booklet should render without errors
